pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -80,31 +79,28 @@ func New(context *clusterd.Context, namespace, version string, placement rookalp
 // Start the mgr instance
 func (c *Cluster) Start() error {
 	logger.Infof("start running mgr")
+	name := instanceName()
```

Comment:
Depending on the outcome of my earlier comment, having this the "same" way as previously to spawn as many replicas as "wanted" by us, would be nice.

Reply:
agreed, my earlier response should cover this